src/main/java/com/gmail/goosius/siegewar/utils/DiscordWebhook.java

```diff
@@ -396,15 +395,21 @@ public static void sendWebhookNotification(Color color, String message) {
     public static void sendWebhookNotification(Color color, String message, boolean active) {
         DiscordWebhook webhook = new DiscordWebhook(SiegeWarSettings.getDiscordWebhookUrl());
 
-        webhook.addEmbed(new DiscordWebhook.EmbedObject()
+        String[] lines = message.split("\n");
+        EmbedObject embed = new DiscordWebhook.EmbedObject()
                 .setColor(color)
-                .setDescription(message)
-        );
+                .setDescription(lines[0]);
+        if (lines.length > 1)
+            for (int i = 1; i < lines.length; i++) {
+                embed.addField("Battle", lines[i], true);
+            }
+        webhook.addEmbed(embed);
+
         try {
             webhook.execute();
         } catch (java.io.IOException e) {
             if (active)
-                SiegeWar.getSiegeWar().getLogger().severe(Arrays.toString(e.getStackTrace()));
+                e.printStackTrace();
```

Comment:
❓ 
- Why was this change made?
- As I understand it, using the logger has better cross-platform support, for example paper seems to like it more than Sysout calls.

Reply:
it was all on one line and really hard to read

Reply:
* You'll have to revert it so please. I don't want angry Paper admins to coming after us.  Those guys are ...... fast..... :-D